packages/aeproject-cli/aeproject-init/artifacts/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     volumes:
       - ./docker/aeternity_node1_mean15.yaml:/home/aeternity/aeternity.yaml
       - ./docker/keys/node1:/home/aeternity/node/keys
-      - node1_db:/home/aeternity/node/data/mnesia
+      # - node1_db:/home/aeternity/node/data/mnesia
```

Comment:
Should we keep those lines as comments for future reference? (concerns node1_db ,node2_db && node3_db)